docusaurus-docs/conda-store/references/faq.md

```diff
@@ -94,22 +94,60 @@ It consists of:
 3. the id of a build
 4. the environment name.
 
-The version 2 format is now the default. Environments created using the version
-1 format will continue to be accessible in the UI, but new builds will use the
-version 2 format. No changes are needed for existing deployments of conda-store.
+However, version 2 build paths don't solve the problem completely because they
+include user-provided data, like the environment name, and that data can be
+arbitrary large.
 
-There is no real reason to use the version 1 format anymore, but it can be
-explicitly set via the config:
+To solve this problem, version 3 was introduced, which will always have the same
+size. It looks like this:
 
-```python
-c.CondaStore.build_key_version = 1
+```bash
+64a943764b70e8fe181643404894f7ae
 ```
 
-The version 2 format can also be explicitly set if needed (this is the same as
-the default):
+It's a truncated SHA-256 hex digest, which is calculated based on:
+
+- namespace name
+- specification hash (also SHA-256)
+- build timestamp
+- build id.
+
+See `BuildKey._version3_fmt` for details.
+
+:::note
+When version 3 is used, `Build.build_path` will not include the namespace name,
+because it's not fixed size, so all builds will be placed right into
+`CondaStore.store_directory`.
+
+Additionally, `CondaStore.environment_directory` will be completely ignored, so
+no symlinks connecting an environment name to its corresponding build will be
+created, because the environment directory format also includes variable-size
+data (the namespace and environment names).
+
+The lack of symlinks doesn't prevent server artifacts from being generated,
+which are available for download via the UI (lockfiles, archives, etc.), because
+those rely on storage or use the database.
+
+But it does impact conda integration or tools that rely on it, like when
+conda-store is used with JupyterLab as part of a Nebari deployment. Without
+environment symlinks, there'll be no way to tell conda where to look for
```

Comment:
Are we creating these symlinks for the end user? Should we? Maybe an example here would be more illustrative.